src/components/App/App.tsx

```diff
@@ -12,32 +12,76 @@ import React, { FC } from "react";
-            <ToastContainer />
-            <Layout>
-                <Flex marginX="size-300" alignItems="start" direction="column" justifyContent="center" gap={"size-200"}>
-                    {profile ? <h3>Welcome {profile?.given_name}</h3> : <></>}
-                    <UsernameInput
-                        label={"Name"}
-                        text={name.textValue}
-                        isBold={name.isBold}
-                        isItalics={name.isItalics}
-                        onInputChange={updateName}
-                    />
-                    {/* <UsernameInput label={"Title"} />
-                    <UsernameInput label={"Company"} />
-                    <UsernameInput label={"Email"} />
-                    <UsernameInput label={"Phone number"} />
-                    <UsernameInput label={"LinkedIn"} /> */}
-                    <ImgInput />
-                    <TablePreview />
-                </Flex>
-            </Layout>
-        </ProviderV3>
+        <AuthenticationProvider>
+            <ProviderV3 theme={isDarkMode ? darkTheme : lightTheme} colorScheme={"light"} height="100%">
+                <ToastContainer />
+                <Layout>
+                    <Flex
+                        marginX="size-300"
+                        alignItems="start"
+                        direction="column"
+                        justifyContent="center"
+                        gap={"size-200"}
+                    >
+                        <UsernameInput
+                            type="name"
+                            label={"Name"}
+                            text={name.textValue}
+                            isBold={name.isBold}
+                            isItalics={name.isItalics}
+                            onInputChange={updateState}
+                        />
+                        <UsernameInput
+                            type="title"
+                            label={"Title"}
+                            text={title.textValue}
+                            isBold={title.isBold}
+                            isItalics={title.isItalics}
+                            onInputChange={updateState}
+                        />
+
+                        <UsernameInput
+                            type="company"
+                            label={"Company"}
+                            text={company.textValue}
+                            isBold={company.isBold}
+                            isItalics={company.isItalics}
+                            onInputChange={updateState}
+                        />
+                        <UsernameInput
+                            type="email"
+                            label={"Email"}
+                            text={email.textValue}
+                            isBold={email.isBold}
+                            isItalics={email.isItalics}
+                            onInputChange={updateState}
+                        />
```

Comment:
We're passing all the variables from the state individually, could we pass it only once to keep the code clean? 
```suggestion
                        <UsernameInput
                            label="Email"
                            state={email}
                            onInputChange={updateState}
                        />
```

Then inside of `UsernameInput` it can do `state.isBold`, `state.type`, and `state.isItalics`